contrib/completion/zsh/_docker

```diff
@@ -2122,7 +2122,7 @@ __docker_subcommand() {
                 "($help)--ipv6[Enable IPv6 networking]" \
                 "($help -l --log-level)"{-l=,--log-level=}"[Logging level]:level:(debug info warn error fatal)" \
                 "($help)*--label=[Key=value labels]:label: " \
-                "($help)--live-restore[Enable live restore of docker when containers are still running]" \
+                "($help)--live-restore[Enable live restore of docker when containers are still running. This option is not supported on Windows.]" \
```

Comment:
Actually, I just realized that this is completion for `dockerd`, so would never be run on a different platform than the daemon (as this is the daemon), so these scripts don't have to mention it's "not supported on Windows"